core/server/api_container/server/startosis_engine/startosis_interpreter.go

```diff
@@ -386,6 +391,16 @@ func (interpreter *StartosisInterpreter) buildBindings(
 	return &predeclared, nil
 }
 
+const (
+	moduleIdSeperator     = "/"
+	numModuleIdSeparators = 4
+	prefixNum             = 3
+)
+
+func getModulePrefix(moduleId string) string {
+	return strings.Join(strings.SplitN(moduleId, moduleIdSeperator, numModuleIdSeparators)[:prefixNum], moduleIdSeperator)
```

Comment:
From what I understand of `SplitN`, isn't `numModuleIdSeparators` always just `prefixNum` + 1 (so we don't need to maintain the extra const)?